### PR TITLE
Add maintenance script to new Interwiki building block

### PR DESCRIPTION
**Pull request by Silvan-WMDE**
This introduces a new "Interwiki" building block to the Client's
Overview diagram and adds a maintenance script to the corresponding
subsection.

The Interwiki building block is expected to be completed later.

Bug: [T272972](https://phabricator.wikimedia.org/T272972)

### CODE REVIEW

**Review by addshore — CHANGES_REQUESTED** [review]
I think the interwiki block within the client should also exist in the context and scope diagram?
This would connect to the Interwiki block in MediaWiki which is also not currently represented there.

**Comment by addshore** [comment]
In systems/WikibaseClient/diagrams/05-building-blocks.drawio.svg this PR introduced an arrow going from Repo -> Interwiki, which in the long term may be correct, but isn't really represented by the current content of the docs (as we haven't finished client yet).
I feel that we should wait until we introduce that arrow with purpose and an idea of how the block works behind it, rather than assuming that repo probably directly interacts with this interwiki block.

This is already reflected in the context diagram, which doesn't have this connection.

**Review by tzhelyazkova — APPROVED** [review]

File: systems/WikibaseClient/05-Building_Block_View.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 | [Linked Site Page Changes](#linked-site-page-changes)       | Inform a Repository of changes to pages that are linked to Repository Items      |
 | [Client Side Item Edits](#client-side-item-edits)           | Edit Item Data on the Repository                                                 |
 | [Special Pages](#special-pages)                             | WikibaseClient specific [Special Pages](../overview/12-Glossary.md#special-page) |
+| [Interwiki](#interwiki)                                     | Client side interactions with Interwiki links                                    |
```
Comment (addshore) [inline systems/WikibaseClient/05-Building_Block_View.md:15]: We say "WikibaseClient specific" above, but "Client side" here.
I believe these are meant to mean the same thing?

**Review by addshore — APPROVED** [review]

**Comment by addshore** [comment]
Left 1 comment, but not blocking getting this merged on that.

**Pull request merged**